src/components/layer-controller/LayerOptions.js

```diff
@@ -42,6 +42,35 @@ function OpacitySlider({ value, handleChange }) {
   );
 }
 
+function GlobalSelectionSlider({
```

Comment:
I think defaulting to a slider with discrete values is fine for now, but would be open to other suggestions.

Reply:
Agreed, there are a discrete number of selections to make. A half-index selection doesn't really make sense?